src/inspector_agent.h

```diff
@@ -86,12 +86,19 @@ class Agent {
   std::unique_ptr<ParentInspectorHandle> GetParentHandle(
       int thread_id, const std::string& url);
 
-  // Called to create inspector sessions that can be used from the main thread.
+  // Called to create inspector sessions that can be used from the same thread.
```

Comment:
Does it mean that I can call `inspector.open(...)` from worker thread?

Reply:
Nothing prevents it for now... Feel free to open a bug if this is something that needs to be fixed.